src/Microsoft.Azure.WebJobs.Host/Config/WebJobsExtensionOptionsConfiguration.cs

```diff
@@ -9,20 +9,24 @@ namespace Microsoft.Azure.WebJobs.Host.Config
 {
     internal class WebJobsExtensionOptionsConfiguration<TOptions> : IConfigureOptions<TOptions> where TOptions : class
     {
-        private readonly string _extensionName;
         private readonly IConfiguration _configuration;
         private readonly Action<IConfiguration, string, TOptions> _configure;
+        private readonly ExtensionInfo _extensionInfo;
 
-        public WebJobsExtensionOptionsConfiguration(IConfiguration configuration, string extensionName, Action<IConfiguration, string, TOptions> configure)
+        public WebJobsExtensionOptionsConfiguration(IConfiguration configuration, ExtensionInfo extensionInfo, Action<IConfiguration, string, TOptions> configure)
```

Comment:
Changes to this file are no longer needed now, are they?

Reply:
Good point. Rollback the change.